modules/DoubleClick/resources/mw.DoubleClick.js

```diff
@@ -1556,6 +1578,16 @@
                 'height': this.embedPlayer.getVideoHolder().height()
             };
         },
+        hideControlsOnVPAID: function(){
+            var hideControls = "-40px";
```

Comment:
what happens if we are with actual hover controls?
also, `-40px` - what if we have custom CSS?

Reply:
Maybe consider moving this to controlBars classes as methods and doing `this.embedPlayer.layoutManager.hideControls()` kind of a thing

Reply:
@OrenMe I checked with hover controls and it works, I can make the pixel size larger as I do not want to mess around with z-index.

Will move it to controlBars